scripts/publish-github-release.js

```diff
@@ -16,7 +16,7 @@ async function publishRelease(octokit, { repo }) {
     'utf-8',
   );
 
-  const versionHeader = /^##/gm;
+  const versionHeader = /^##(?!#)/gm;
```

Comment:
Would something like this work? (allowlist instead of denylist)

```suggestion
  const versionHeader = /^## \d+\.\d+\.\d+/gm;
```

Reply:
Ye, this would work as well - I just didn't want to rely on the exact spacing between hashes and the version number. But you are probably right - this looks more straightforward so it will be easier to understand in the future.